extension/ts/toast.ts

```diff
@@ -0,0 +1,69 @@
+export enum ToastType {
+	WARNING,
+	SUCCESS,
+}
+
+function styleInject(cssText: string) {
+	const head = document.head || document.getElementsByTagName("head")[0];
+	const style = document.createElement("style");
+	style.appendChild(document.createTextNode(cssText));
+	head.appendChild(style);
+	return style;
+}
+
+const createToast = (toastType: ToastType) => {
+	const toast = document.createElement("div");
+	toast.id = `better-library-thing-toast-${Math.floor(Math.random() * 100)}`;
```

Comment:
to handle multiple toasts at the same time (conflicts with the generated number, if any, should be rare)

Reply:
Actually, couldn't you just increment a counter? The counter "resets" when you navigate away from the page or reset.